.github/workflows/release_publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish-npm:
+    # Only run if release is a prod release
+    if: "!github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          ref: refs/heads/master
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+          check-latest: true
+      - name: npm install
+        run: npm install
+      - name: version and publish
+        run: |
+          export MAJOR_MINOR_SEMVER=${GITHUB_REF:11}
+          git config user.name $GITHUB_ACTOR
+          git config user.email gh-actions-${GITHUB_ACTOR}@github.com
+          git remote add gh-origin https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+
+          npm version ${MAJOR_MINOR_SEMVER} --force -m "Bump version to %s"
+          git push gh-origin HEAD:master --tags
+
+          npm publish --access public
```

Comment:
Does this need any other credentials to push this?

Reply:
@cejohnson-bw Didn't say that he needed any when he gave me the commands, but he might have set up a token and a local .npmrc file

Reply:
I got a token from DevX and added it as a repository secret. Will update the script accordingly...